agave_app/renderer.cpp

```diff
@@ -64,11 +62,11 @@ Renderer::init()
   // QMessageBox::information(this, "Info:", "Application Directory: " + QApplication::applicationDirPath() + "\n" +
   // "Working Directory: " + QDir::currentPath());
 
-  QSurfaceFormat format = renderlib::getQSurfaceFormat();
-
-  this->m_glContext = new QOpenGLContext();
-  this->m_glContext->setFormat(format); // ...and set the format on the context too
-  this->m_glContext->create();
+#if HAS_EGL
```

Comment:
Just for my understanding: you're using compiler directives(?)--here and elsewhere in this diff--to ensure that only certain branches of code get compiled on different platforms?

Reply:
That's correct.  The HAS_EGL flag only gets defined if compilation is happening on Linux.   All those `#if`s get preprocessed before compile time and as a result only the HAS_EGL side gets compiled into the linux build.